misc/debug-adapter/modules/debug-adapter-core/src/main/java/org/ballerinalang/debugadapter/evaluation/engine/FunctionInvocationExpressionEvaluator.java

```diff
@@ -91,8 +91,7 @@ public BExpressionValue evaluate() throws EvaluationException {
         SemanticModel semanticContext = context.getDebugCompiler().getSemanticInfo();
         List<Symbol> functionMatches = semanticContext.moduleSymbols()
                 .stream()
-                .filter(symbol -> symbol.kind() == SymbolKind.FUNCTION
-                        && modifyName(symbol.getName().get()).equals(functionName))
+                .filter(symbol -> symbol.kind() == FUNCTION && modifyName(symbol.getName().get()).equals(functionName))
```

Comment:
Will `symbol.getName()` always present? Shall we add `isPresent` check for `symbol.getName()`?
```suggestion
                .filter(symbol -> symbol.kind() == FUNCTION && symbol.getName().isPresent() && modifyName(symbol.getName().get()).equals(functionName))
```

Reply:
+1. Added in 8a8a248